### ISSUE TITLE
Local Apache Pulsar Environment 

### ISSUE THREAD

**Issue by MPSebastian** [amosproj/amos2023ss01-apache-pulsar-ui#20]
## User story
1. As a Developer
2. I need a functional apache pulsar instance with some dummy data
3. So that I have an endpoint, where I can test my implementations.

## Acceptance criteria
* [ ] Create an environment, where an apache pulsar instance is running
* [ ] The apache pulsar instance is filled with dummy data (topic, messages)
* [ ] The instance has to be usable for every developer

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by nikkite99** [comment]
Topics, Producer, Consumer and Messages
